packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -50,41 +50,40 @@
 }
 
 .font-library-modal__font-card {
-	border: 1px solid #e5e5e5;
+	border: 1px solid $gray-200;
 	width: 100%;
 	height: auto;
-	padding: 1rem;
+	padding: $grid-unit-20;
 	margin-top: -1px; /* To collapse the margin with the previous element */
 
 	.font-library-modal__font-card__name {
 		font-weight: bold;
 	}
 
 	.font-library-modal__font-card__count {
-		color: #6e6e6e;
+		color: $gray-700;
 	}
 }
 
 .font-library-modal__library-font-variant {
-	border: 1px solid #e5e5e5;
-	padding: 1rem;
+	border: 1px solid $gray-200;
+	padding: $grid-unit-20;
 	margin-top: -1px; /* To collapse the margin with the previous element */
 }
 
 .font-library-modal__font-variant {
-	border-bottom: 1px solid #e5e5e5;
-	padding-bottom: 1rem;
+	border-bottom: 1px solid $gray-200;
+	padding-bottom: $grid-unit-20;
 }
 
 .font-library-modal__tabs {
 	[role="tablist"] {
 		position: sticky;
 		top: 0;
-		width: calc(100% + 64px);
```

Comment:
There is a negative margin on the left and right, so there is no need to calculate the width.